test/optimizations/deadCodeElimination/elliot/countDeadModules.good

```diff
@@ -1 +1 @@
-Removed 33 dead modules.
+Removed 32 dead modules.
```

Comment:
This is strange to me, why did adding a submodule (presumably the one in Search) mean there are now less dead modules?

Reply:
I suspect it's directly related to why Search is turning up in the other module initialization tests - I think the no-longer-dead module is Search.